src/frontend/sass/splash-screen.scss

```diff
@@ -0,0 +1,44 @@
+@import './mat-colors';
+// Custom theme support
+@import './custom';
+app-root {
```

Comment:
We shouldn't be applying styles here as these are going to be applied application-wide. Adding `display:flex` here is causing issues once logged into the application.
<img width="1680" alt="screen shot 2018-05-31 at 16 37 51" src="https://user-images.githubusercontent.com/2823726/40792169-2dcb1728-64f1-11e8-99e2-e1920b4cb68c.png">